src/restaurant_inspections_api/handlers/businesses.clj

```diff
@@ -0,0 +1,55 @@
+(ns restaurant-inspections-api.handlers.businesses
```

Comment:
Yeah, there are also some logic that could be generalized like generating the validation dictionary